src/validators/validate.js

```diff
@@ -356,13 +140,12 @@ export const walkValidationTree = data => {
   for (const property in data) {
     // When the property is not specific to this instance
     // skip it and go to the next.
-    if (!data.hasOwnProperty(property)) {
-      continue
-    }
 
-    const result = walkValidationTree(data[property])
-    if (!result) {
-      return false
+    if (!data[property]) {
```

Comment:
This looks like it should be `if (data[property]) {...` since it's accessing that value?